src/Settings.js

```diff
@@ -1,10 +1,20 @@
+// @flow
 /**
  * This is a module for storing settings passed into KaTeX. It correctly handles
  * default settings.
  */
 
 import utils from "./utils";
 
+type SettingsOptions = {
+    displayMode?: boolean;
+    throwOnError?: boolean;
+    errorColor?: string;
+    macros?: {[macroName: string]: string};
+    colorIsTextColor?: boolean;
+    maxSize?: number;
```

Comment:
This could be `maxSize: number` since we always set `maxSize` to a number in the constructor.

Reply:
I'm not sure I'm following. `SettingsOptions` is what the user passes into the `Settings` constructor, and as we established earlier, all the properties here should be optional.

Reply:
Sorry, my bad.  I'm confounding `SettingOptions` with the properties on the `Settings` class.